examples/todomvc/components/TodoTextInput.js

```diff
@@ -0,0 +1,49 @@
+import React, { Component, PropTypes } from 'react';
+import classnames from 'classnames';
+
+export default class TodoTextInput extends Component {
+  static propTypes = {
+    handleSave: PropTypes.func.isRequired,
```

Comment:
I hadn't the time to check this before it has been merged.

The props referring to events should be named `on[EventName]`.
The handlers should be named `handle[EventName]`.

Can you rename it to `onSave`?